planning/mission_planner/src/lanelet2_plugins/default_planner.hpp

```diff
@@ -36,6 +36,7 @@ namespace mission_planner::lanelet2
 struct DefaultPlannerParameters
 {
   double goal_angle_threshold_deg;
+  bool correct_goal_pose;
```

Comment:
I'm not sure about the best way to name the boolean flag, but the flag name which starts with a verb sounds a bit weird.

According to https://www.samanthaming.com/tidbits/34-better-boolean-variable-names/, enable/disable + verb may be better?

Reply:
done :+1: